### ISSUE TITLE
Accessibility - Line height is below minimum value

### ISSUE THREAD

**Issue by id2win** [GSA/idmanagement.gov#540]
Issue level AAA

Why?
When lines of text are too close together, the text becomes harder to read. This can affect the usability of your site for all visitors — but especially those with low vision or cognitive conditions such as dyslexia.

Line height recommendation is 1.5 times the font size to enable better readability. Details along with specific location for 4 occurrences found by visiting 

- [x] https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?PageId=38246420896&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r73/failed/MDlLNElwcXQ=;Y2Y2MzBiNmI=

**Comment by claytonjbarnette** [comment]
Resolved in  PR #542
